feat(docs): added notes on OpenShift (#504)
stackabletech/documentation: modules/ROOT/pages/kubernetes.adoc

Consult the xref:release_notes.adoc[release notes] to find out which specific versions are supported for the Stackable Data Platform you are using.

[#openshift-notes]
== Notes on OpenShift

> SDP operators are certified for the OpenShift platform ans can be installed from the OperatorHub.

Reviewer: ```suggestion
SDP operators are certified for the OpenShift platform and can be installed from the OperatorHub.
```

> NOTE: Install the SDP operators in a namespace/project called `stackable-operators`. Installing the operators anywhere else may lead to unwanted efects of even faulty installations. This restriction is caused by limitations in the Operator Bundle Format (OPM) used by OpenShift.

Reviewer: ```suggestion
NOTE: Install the SDP operators in a namespace/project called `stackable-operators`. Installing the operators anywhere else may lead to unwanted effects or even faulty installations. This restriction is caused by limitations in the Operator Bundle Format (OPM) used by OpenShift.
```

=== Customizing operator installations
Depending on the cluster size, you may need to customize the resources requested by the SDP operator Pods. This is possible when installing the operators from the command line. For example, to assign `256Mi` of memory to the Apache Kafka operator, you need to create a custom Subscription as follows:

[source,yaml]
----
---
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: stackable-kafka-operator
  namespace: stackable-operators
spec:
  channel: stable
  installPlanApproval: Automatic
  name: stackable-kafka-operator
  source: certified-operators
  sourceNamespace: openshift-marketplace
  startingCSV: kafka-operator.v23.11.0
  config:
    resources:
      limits:
        memory: 256Mi
      requests:
        memory: 256Mi
----

> In addition, You can restrict the operator to watch a specific namespace. By default, the operator watch all namespaces. For example, to restrict the Kafka operator to watching only a namespace called `kafka-namespace` you add the following properties to the Subscription manifest:

Reviewer: ```suggestion
In addition, You can restrict the operator to watch a specific namespace. By default, the operator watches all namespaces.
For example, to restrict the Kafka operator to watching only a namespace called `kafka-namespace` you add the following properties to the Subscription manifest:
```
